libraries/cbor/src/values.rs

```diff
@@ -183,6 +183,22 @@ where
     }
 }
 
+impl From<Vec<(KeyType, Value)>> for Value {
+    fn from(map: Vec<(KeyType, Value)>) -> Self {
+        Value::Map(map)
+    }
+}
+
+impl From<BTreeMap<KeyType, Value>> for Value {
```

Comment:
Is this temporary until all BTreeMap occurrences are removed or are there places where we plan to keep using BTreeMap?

Reply:
I already removed all calls using `BTreeMap`. I didn't see a reason why our library API shouldn't support initializing from `BTreeMap`. Since it's unused, it doesn't affect binary size.

Reply:
Ok I see, it's from the point of view of CBOR as a library that is not only used by OpenSK. Sounds good to me, although the library is still optimized for OpenSK use-case.